libs/env/src/env.ts

```diff
@@ -553,6 +553,11 @@ const envDefinitions = {
     parser: boolParser,
     desc: "dev flag to skip onboarding flow",
   },
+  SPECULOS_API_PORT: {
+    def: 0,
```

Comment:
Default value should be something like 40000 no ?

Reply:
indeed